src/Writer/CsvWriter.php

```diff
@@ -58,21 +58,28 @@ class CsvWriter implements TypedWriterInterface
      */
     protected $withBom;
 
+    /**
+     * @var string
+     */
+    protected $terminate;
+
     /**
      * @param string $filename
      * @param string $delimiter
      * @param string $enclosure
      * @param string $escape
      * @param bool   $showHeaders
      * @param bool   $withBom
+     * @param string $terminate
      */
-    public function __construct($filename, $delimiter = ',', $enclosure = '"', $escape = '\\', $showHeaders = true, $withBom = false)
+    public function __construct(string $filename, string $delimiter = ',', string $enclosure = '"', string $escape = '\\', bool $showHeaders = true, bool $withBom = false, string $terminate = "\n")
```

Comment:
We support php 5.6, please remove string typehints

Reply:
Bool also

Reply:
Also, this line is longer than 120 chars, please break it

Reply:
:+1: